components/x-interaction/src/concerns/wrap-component-name.js

```diff
@@ -1,7 +1,5 @@
-import { getComponentName } from './register-component'
-
 function wrapComponentName(Component, Enhanced) {
-	const originalDisplayName = getComponentName(Component)
+	const originalDisplayName = Component.displayName || Component.name
```

Comment:
is `Component.name` available? since when the component is getting registered it does `Component[xInteractionName] = name`

Reply:
`Component.name` here is the built-in Javascript function `name` so it should be there, `wrapComponentName` is only for debugging purposes, so it doesn't really matter :)